hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2587,6 +2587,42 @@
       tests: []
     }
 
+    //////////////////
+    // CPU          //
+    // RV_CORE_IBEX //
+    //////////////////
+
+    {
+      name: chip_sw_rv_core_ibex_rnd
+      desc: '''Verify the functionality of the random number generation CSRs.
+
+               - Perform multiple reads from `RND_DATA` polling `RND_STATUS` in
+                 between to only read valid data. Check different random bits
+                 are provided each time.
+               - Ensure `RND_STATUS` indicate invalid data immediately after
+                 `RND_DATA` read.
+               - Perform repeated reads from `RND_DATA` without `RND_STATUS`
+                 polling to check read when invalid doesn't block.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_sw_rv_core_ibex_address_translation
+      desc: '''Verify the simple address translation functionality.
+
+               - Setup address translation for both I and D side and check
+                 correct translation for I and D accesses.
```

Comment:
There's two slots for I and D, I think we should mention both here to be tested.
Maybe we should have a step to test the feature:
`If a transaction matches multiple regions, the lowest indexed region has priority.`